### PR TITLE
feat(opencollective prompt): add disable option

### LINKED ISSUE #854: [request] opencollective postinstall should respect DISABLE_OPENCOLLECTIVE
**Is your feature request related to a problem? Please describe.**
The inhouse opencollective postinstall `bin/opencollective.js` should respect DISABLE_OPENCOLLECTIVE (or CI) environment variable just like standard opencollective-postinstall.

https://github.com/opencollective/opencollective-postinstall/blob/7a5c767e043f7e86c452f5f8b96b9cf4db7a8ce5/index.js#L7

**Describe the solution you'd like**
It should respect environment variables DISABLE_OPENCOLLECTIVE and CI, in order to allow user to turn the donation prompt off.

**Describe alternatives you've considered**
NA

**Additional context**
NA

### PR DESCRIPTION
**Pull request by misterdev**
**What kind of change does this PR introduce?**
Allows disabling the open collective prompt.

Following [opencollective-postinstall](https://github.com/opencollective/opencollective-postinstall/blob/master/index.js#L7), our prompt is disabled if `DISABLE_OPENCOLLECTIVE` or `CI` env variables are set to `true`

**If relevant, did you update the documentation?**
Yes

**Summary**
Closes #854 

**Other information**
This is an alternative PR to #856

### CODE REVIEW

File: bin/opencollective.js [hunk 1]
```diff
@@ -30,4 +30,9 @@ function printBadge() {
 	console.log("\n");
 }
 
-printBadge();
+function isTrue(value) {
+	return !!value && value !== "0" && value !== "false";
+}
+var envDisable = isTrue(process.env.DISABLE_OPENCOLLECTIVE) || isTrue(process.env.CI);
```
Comment (pranshuchittora) [inline bin/opencollective.js]:
```suggestion
const envDisable = isTrue(process.env.DISABLE_OPENCOLLECTIVE) || isTrue(process.env.CI);
```

File: bin/opencollective.js [hunk 2]
```diff
@@ -30,4 +30,9 @@ function printBadge() {
...
+
+if (!envDisable) printBadge();
```
Comment (pranshuchittora) [inline bin/opencollective.js]:
```suggestion
envDisable?null : printBadge();
```

Comment (ematipico) [reply bin/opencollective.js]: The ternary check is used to return a value, here we don't need to return anything

File: README.md [hunk 3]
```diff
@@ -91,3 +92,9 @@ The webpack family welcomes any contributor, small or big. We are happy to elabo
 
 [deps]: https://img.shields.io/david/webpack/webpack.svg
 [deps-url]: https://david-dm.org/webpack/webpack-cli
+
+## Open Collective
+
+If you like **webpack** Please consider donating to our [Open Collective](https://opencollective.com/webpack) to help us maintain it.
```
Comment (ematipico) [inline README.md]: `to maintain it,`

File: bin/opencollective.js (hunk 2 above)
Comment (misterdev) [reply bin/opencollective.js]: I prefer the statement, is more readable and we don't need an expression :)

File: README.md [hunk 4]
```diff
@@ -91,3 +92,9 @@ The webpack family welcomes any contributor, small or big. We are happy to elabo
...
+
+We show this message in the terminal once in a while, if you want you can disable it by setting the environment variable `DISABLE_OPENCOLLECTIVE=true` or `CI=true`.
```
Comment (ematipico) [inline README.md]: `once a week`

**Comment by ematipico** [comment]
Could you rebase please? So we can a CI working and merge it smoothly

File: README.md [hunk 5]
```diff
@@ -91,3 +93,9 @@ The webpack family welcomes any contributor, small or big. We are happy to elabo
 
 [deps]: https://img.shields.io/david/webpack/webpack.svg
 [deps-url]: https://david-dm.org/webpack/webpack-cli
+
+## Open Collective
+
+If you like **webpack** Please consider donating to our [Open Collective](https://opencollective.com/webpack) to help us to maintain it.
```
Comment (rishabh3112) [inline README.md]:
```suggestion
If you like **webpack** , please consider donating to our [Open Collective](https://opencollective.com/webpack) to help us maintain it.
```

File: bin/opencollective.js [hunk 6]
```diff
@@ -24,10 +24,15 @@ function printBadge() {
 	console.log("\n");
 	print(`${chalk.bold("Thanks for using")} ${chalk.bold.blue("Webpack!")}`);
 	print(`Please consider donating to our ${chalk.bold.blue("Open Collective")}`);
-	print("to help us maintain this package.");
+	print("to help us to maintain this package.");
```
Comment (rishabh3112) [inline bin/opencollective.js]:
```suggestion
	print("to help us maintain this package.");
```

Comment (misterdev) [reply bin/opencollective.js]: Thanks, see [here](https://github.com/webpack/webpack-cli/pull/863#discussion_r279170165)

Comment (rishabh3112) [reply bin/opencollective.js]: To me w/o `to` (that I suggested to remove) makes more sense still as we have already used `to` before `help` and adding another `to` is redundant. Rest on you @misterdev :)

Comment (misterdev) [reply bin/opencollective.js]: Changed, I prefer your version too :)

**Comment by ematipico** [comment]
We can merge it after we resolve the conflicts!

**Pull request merged**